src/lib/data/html/speeddial.html

```diff
@@ -528,20 +608,31 @@
         $('#fadeOverlay').click();
 });
 
-function emitChanged(pages)
-{
+function searchToggle() {
+    if ("%SEARCH-ENABLED%" == '1') {
+        external.speedDial.setsearchEnabled(false);
```

Comment:
This can be just `external.speedDial.setsearchEnabled("%SEARCH-ENABLED%" == '1')`.

Actually, `%SEARCH-ENABLED%` should be bool and not string.

Reply:
```
function searchToggle() {
    if ("%SEARCH-ENABLED%" == 'true') {
        external.speedDial.setSearchEnabled(false);
    } else {
        external.speedDial.setSearchEnabled(true);
    };
    location.reload();
}
```